### ISSUE TITLE
double check http cache interactions with SW

### ISSUE THREAD

**Issue by wanderview** [mozilla/remote-newtab#152]
Please see this long twitter thread about some gotchas with the SW cache interacting with the http cache:

  https://twitter.com/jeffposnick/status/671451613224001538

I'm not sure these apply here, but I wanted to make sure you guys were aware of it.

**Comment by marcoscaceres** [comment]
Yeah, saw that. Was hoping to chat with you in Orlando about this and the overall design.
